### ISSUE TITLE
Dropout

### ISSUE THREAD

**Issue by gabrevaya** [aaxwaz/Multivariate-Time-Series-forecast-using-seq2seq-in-TensorFlow#6]
Thank you so much for your clear explanations and codes! I'm just starting with tensorflow and all this is very helpful. I have a doubt: are you using any dropout? Because always when setting the parameters for training the model you define KEEP_RATE = 0.5, but I don't see that it is being used anywhere.
If the dropout is not implemented, how should I do it?

**Comment by aaxwaz** [comment]
I am glad you like it. 

Thanks a lot for pointing that out! You are right that I am not using dropout in my model of seq2seq. 

If you really want to implement that, you can add a dropout layer between input layer and lstm cell block at each time step. This way, for a given dropout probability, there is certain chance that some of the input dimension will be masked as zeros during training. However, I guess this may work well only on data that has a much larger dimension. 

You can also try to add in dropout layer between each lstm cell (of two consecutive time steps), which is called recurrent dropout. However, I never tried this dropout in tf and you may google for the solution if you like.
